File: pset1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int altura = 0;
    do
    {
        altura = get_int("altura: ");
    }
    while (altura <= 0 || altura > 8);

    for (int i = 1; i <= altura; i++)
    {
        for (int spaces = 0; spaces < altura - i; spaces++)
        {
            printf(" ");
        }
        for (int hash = 0; hash < i; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    a = get_int("heigth: ")
    if a >= 1 and a <= 8:
        break
for e in range(a):
    print(" " * (a - 1 - e), end="")
    print("#" * (e + 1), end="")
    print(" " * 2, end="")
    print("#" * (e + 1), end="")
    print(end="\n")